src/NUnitFramework/framework/nunit.framework-4.5.csproj

```diff
@@ -291,6 +294,9 @@
     <Compile Include="Constraints\UniqueItemsConstraint.cs" />
     <Compile Include="Constraints\XmlSerializableConstraint.cs" />
     <Compile Include="Contains.cs" />
+    <Compile Include="CsvData.cs" />
```

Comment:
Obviously, you will need to add these to all of the other targets. Probably easier to do once you merge latest from master.